ui/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -268,30 +277,57 @@ export default class PermissionConnect extends Component {
   }
 
   renderTopBar() {
-    const { redirecting } = this.state;
+    const {
+      redirecting,
+      ///: BEGIN:ONLY_INCLUDE_IN(snaps)
+      targetSubjectMetadata,
+      ///: END:ONLY_INCLUDE_IN
+    } = this.state;
     const { page, isRequestingAccounts, totalPages } = this.props;
     const { t } = this.context;
     return redirecting ? null : (
-      <div className="permissions-connect__top-bar">
-        {page === '2' && isRequestingAccounts ? (
-          <div
-            className="permissions-connect__back"
-            onClick={() => this.goBack()}
-          >
-            <Icon
-              name={IconName.ArrowLeft}
-              marginInlineEnd={1}
-              size={IconSize.Xs}
+      <Box
+        style={{
+          ///: BEGIN:ONLY_INCLUDE_IN(snaps)
+          marginBottom:
+            targetSubjectMetadata.subjectType === SubjectType.Snap && '14px',
```

Comment:
this should be moved out of `style` in favour of Box `marginBottom` own prop and use either `2` or `4` for the value

Reply:
After looking at it it seems like with the legacy style it's too much pain to make this change.